### DISCUSSION TITLE
Is my work good ?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#102571] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I have always wanted to know what  people think of my projects most  of  my friends and discord servers are either toxic or too nice

so Id like to know what  you guys think https://github.com/FFUV?tab=repositories

**Comment by onurravli** [comment]
Hello, I suggest you to add description to your repositories. Some of your repositories doesn't have a description, it'll be more descriptive for users.

**Reply by ghost** [reply]
can you tell me some repristories that dont have

**Reply by onurravli** [reply]
![image](https://github.com/community/community/assets/47084109/2b3f1e2c-4879-48bc-9b94-02b453bb48b3)
![image](https://github.com/community/community/assets/47084109/c4114355-01ef-4ce5-a83e-c25e6acca46c)

**Comment by ghost** [comment, marked as answer]
Hey @FFUV looks like your projects are almost at pro level but it still needs some attention.

- give the url for your projects that are hosted in cloud.
- improve project file structure by moving all project files inside src or app folder(whatever you want to name)
- use ci/cd to fasten up your project routine
- use emojis in your README(trust me this works) 
- Add essential files like License, contributing.md, code files (guess you would have did these already)

Thats all I could suggest.

Happy Programming Journey :)

Thank you
